app/src/main/java/org/oppia/android/app/player/audio/AudioFragmentPresenter.kt

```diff
@@ -113,6 +120,26 @@ class AudioFragmentPresenter @Inject constructor(
     return binding.root
   }
 
+  private fun startSpotlights() {
+    val audioLanguageIconView = binding.audioLanguageIcon
+    audioLanguageIconView.doOnPreDraw {
```

Comment:
onPreDraw needed here?

Reply:
yes, since the slide down animation is being used, the spotlight works correctly only with this despite of the dynamic anchor. Moved this logic to spotlgiht fragment though.

Reply:
It seems that the doOnPreDraw bit was removed in the latest changes.